configs/config.dell_optiplex_9010.uefi

```diff
@@ -0,0 +1,28 @@
+# CONFIG_DRIVERS_INTEL_WIFI is not set
+# CONFIG_PXE_SERIAL_CONSOLE is not set
+CONFIG_LOCALVERSION="v0.1.0"
+CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_BOOTSPLASH_COMPRESS_LZMA=y
+CONFIG_BOOTSPLASH_FILE="dasharo-black.bmp"
+CONFIG_BOOTSPLASH_IMAGE=y
```

Comment:
@miczyg1 do we need `CONFIG_BOOTSPLASH_*` options when we use `CONFIG_EDK2_BOOTSPLASH_*` ? Does it give any value?